### PR TITLE
[WIKI] Fix command formatting, broken links OSB Activites

### PR DESCRIPTION
**Pull request by wontonstrips**
### Description:

Review and fix command formatting and broken links in /docs/src/content/docs/osb/Activites

### Changes:

Fix command formatting aerial-fishing.md
Fix formatting, broken links fountain-of-rune.md
Fix typo spacing guardians-of-the-rift.md
Fix command formatting hallowed-sepulchre.md
Fix command formatting mage-training-arena.md
Fix command formatting mahogany-homes.md
Add command, fix broken links motherlode-mine.md
Fix command formatting shooting-stars.md
Fix command formatting, broken link volcanic-mine.md

Fix broken link wintertodt.md

### Other checks:

- [ x] I have tested all my changes thoroughly.

### CODE REVIEW

File: docs/src/content/docs/osb/Activities/wintertodt.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can train Firemaking via the Wintertodt minigame using the command below. Yo
 
 ## Warm Clothing
 
-While optional, it is highly advised to equip **at least 4 pieces** of warm clothing **in your skilling setup**to reduce your food consumption by up to 37.5%. This will take your food consumption down from 160hp/kill, to 100hp/kill. The easiest warm gear you can obtain on the bot is Clue hunter, to do this see [Crack The Clue](../../miscellaneous/crack-the-clue.md). Warm clothes can be a combination of anything from the following list:
+While optional, it is highly advised to equip **at least 4 pieces** of warm clothing **in your skilling setup**to reduce your food consumption by up to 37.5%. This will take your food consumption down from 160hp/kill, to 100hp/kill. The easiest warm gear you can obtain on the bot is Clue hunter, to do this see [Crack The Clue](https://wiki.oldschool.gg/osb/miscelleanous/crack-the-clue/). Warm clothes can be a combination of anything from the following list:
```
[the remark this answers is not in the document]
Comment (wontonstrips) [reply docs/src/content/docs/osb/Activities/wintertodt.md:23]: 'miscelleanous' is the correct spelling

File: docs/src/content/docs/osb/Activities/volcanic-mine.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ You can do the Volcanic Mine using: `/minigames volcanic_mine start`
 Boosts must be equipped in the skilling setup.
 
 - 30% more points & XP for 61+ mining and dragon pickaxe
-- 50% more points & XP for 71+ mining and [crystal pickaxe](zalcano.md)
+- 50% more points & XP for 71+ mining and [crystal pickaxe](https://wiki.oldschool.gg/osb/miscelleanous/zalcano/)
```
[the remark this answers is not in the document]
Comment (wontonstrips) [reply docs/src/content/docs/osb/Activities/volcanic-mine.md:26]: 'miscelleanous' is the correct spelling

**Comment by DayV-git** [comment]
At this point just rename the misc folder so it's spelt correctly. If you do, change the zalcano link in k.ts

**Comment by wontonstrips** [comment]
Can do but will wait until I have some time to fix lots of links after making that change.  Admin can merge this PR if they want or wait i'll leave that decision to them.

**Pull request merged**